### ISSUE TITLE
Support for FailoverPartner in Connection config?

### ISSUE THREAD

**Issue by hajikelist** [LeanKit-Labs/seriate#64]
Can't seem to find any mention of failover partner configuration, is this feature supported in seriate?

**Comment by hajikelist** [comment]
...failover is pretty mission critical for a lot of implementations... be nice to see this feature addressed.

**Comment by mpayne-sds** [comment]
No-one needs failover anymore. Everything works perfectly all the time, especially MS SQL Server. /sarcasm
